<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QWidget" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QGroupBox" name="parametersBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>164</width>
     <height>190</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Parameters</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="baudRateLabel">
      <property name="text">
       <string>BaudRate:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="baudRateBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dataBitsLabel">
      <property name="text">
       <string>Data bits:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="dataBitsBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="parityLabel">
      <property name="text">
       <string>Parity:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="parityBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="stopBitsLabel">
      <property name="text">
       <string>Stop bits:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="stopBitsBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="flowControlLabel">
      <property name="text">
       <string>Flow control:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="flowControlBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="selectBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>0</y>
     <width>101</width>
     <height>190</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Serial Port</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="serialPortInfoListBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="manufacturerLabel">
      <property name="text">
       <string>Manufacturer:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="serialNumberLabel">
      <property name="text">
       <string>Serial number:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="locationLabel">
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="vidLabel">
      <property name="text">
       <string>Vendor ID:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="pidLabel">
      <property name="text">
       <string>Product ID:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="additionalOptionsGroupBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>196</y>
     <width>271</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>Additional options</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="localEchoCheckBox">
      <property name="text">
       <string>Local echo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
